### PR TITLE
A-Team updates: new phenotypes, multiple genetic modifiers, etc.

### PR DESCRIPTION
**Pull request by chris-grove**
- Added several new ontology term classes
- Added 'mod_internal_id' slot definition; slot added to DiseaseAnnotation class
- Added 'synonyms' slot to VocabularyTerm class; updated description of 'synonyms' slot in core.yaml
- Changed 'disease_genetic_modifier' slot to 'disease_genetic_modifiers' and made multivalued

### CODE REVIEW

File: model/schema/core.yaml [hunk 1]
```diff
@@ -857,6 +858,11 @@ slots:
       The model organism database (MOD) identifier/curie for the object
     range: string
 
+  mod_internal_id:
+    description: >-
+      The model organism database (MOD) internal identifier for the object
+      range: string
```
Comment (markquintontulloch) [inline model/schema/core.yaml:864]: indentation is off here

File: model/schema/phenotypeAndDiseaseAnnotation.yaml [hunk 2]
```diff
@@ -183,6 +183,7 @@ classes:
       - curie
       - unique_id
       - mod_entity_id
+      - mod_internal_id
```
Comment (markquintontulloch) [inline model/schema/phenotypeAndDiseaseAnnotation.yaml:186]: Need to add this to the DTO class too

File: model/schema/ontologyTerm.yaml [hunk 3]
```diff
@@ -286,9 +285,51 @@ classes:
...
+      PURL: http://purl.obolibrary.org/obo/APO.owl
+
+  DPOTerm:
+    is_a: PhenotypeTerm
+    description: >-
+      A phenotype ontology term from the Drosophila Phenotype Ontology (DPO).
+      PURL: http://purl.obolibrary.org/obo/dpo.owl
+
+  PATOTerm:
+    is_a: OntologyTerm
```
Comment (markquintontulloch) [inline model/schema/ontologyTerm.yaml:323]: Should this be PhenotypeTerm?  Maybe traits aren't phenotypes but it feels like we probably want to use this ontology in fields that are restricted to phenotype terms.

Comment (chris-grove) [reply model/schema/ontologyTerm.yaml:323]: @markquintontulloch  I am not aware of any curators using PATO terms directly for phenotypes. These terms tend to be used to help generate logical definitions for phenotype terms, but are not used directly as phenotype terms

Comment (chris-grove) [reply model/schema/ontologyTerm.yaml:323]: ...but I can double check with curators

Comment (markquintontulloch) [reply model/schema/ontologyTerm.yaml:323]: cool, we can always change later

**Review by markquintontulloch — APPROVED** [review]
Looks good to me

**Comment by chris-grove** [comment]
@markquintontulloch 
Great! Thanks! I'm just going to add an example 'mod_internal_id' to one of the test files and then I'll be done. All tests pass.

File: model/schema/controlledVocabulary.yaml [hunk 4]
```diff
@@ -91,10 +91,15 @@ classes:
...
       - vocabulary_description
       - member_terms
+      - synonyms
     slot_usage:
       name:
         required: true
         identifier: true
+      synonyms:
+        description: >-
+          Free text synonym(s) of a controlled vocabulary term
```
Comment (sierra-moxon) [inline model/schema/controlledVocabulary.yaml:101]: is this supposed to be on "Vocabulary"?  the description talks about vocabulary `term` ?

Comment (chris-grove) [reply model/schema/controlledVocabulary.yaml:101]: @sierra-moxon Good catch! Thanks! Fixed

**Pull request merged**